$OpenBSD$

Now j.root-servers.net is 192.158.128.30. (SERIAL 2002110501)
Remove a.root-servers.net for overload.

Use m.root-servers.net first for Japenese user.

--- dnsroots.global.orig	Thu Jul  3 22:34:58 2003
+++ dnsroots.global	Thu Jul  3 22:35:58 2003
@@ -1,4 +1,4 @@
-198.41.0.4
+202.12.27.33
 128.9.0.107
 192.33.4.12
 128.8.10.90
@@ -7,7 +7,6 @@
 192.112.36.4
 128.63.2.53
 192.36.148.17
-198.41.0.10
+192.58.128.30
 193.0.14.129
 198.32.64.12
-202.12.27.33
